autoload/db_ui/dbout.vim

```diff
@@ -153,3 +180,4 @@ function! s:get_cell_line_number(scheme) abort
 
   return a:scheme.cell_line_number
 endfunction
+" vim:ts=2:expandtab:
```

Comment:
I got a little frustrated changing my config settings to match the convention, so I added a modeline. Hopefully you don't mind :$

Reply:
We could set up an .editorconfig file.

Reply:
That is probably more comprehensive than a modeline.

I'll remove the modeline and we can add an editorconfig :+1: